src/app/lib/utilities/getRadioSchedulesUrls/index.jsx

```diff
@@ -0,0 +1,16 @@
+export const getRadioScheduleEndpoint = ({
+  service,
+  radioService = service,
+  env,
+  queryString,
+}) => {
+  if (env !== 'live' && queryString) {
+    return `/${service}/bbc_${radioService}_radio/schedule.json${queryString}`;
+  }
+  return `/${service}/bbc_${radioService}_radio/schedule.json`;
```

Comment:
const query = env !== 'live' && queryString ? queryString : '';
return `/${service}/bbc_${radioService}_radio/schedule.json${query}`;

Reply:
@DenisHdz Thanks, I've done this now.